### PR TITLE
android local evaluation

### PR DESCRIPTION
**Pull request by bgiori**
# Amplitude Developer Docs PR

## Description

Local evaluation documentation for android. 

## Deadline

Today

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [x] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [x] My documentation follows the style guidelines of this project.
- [x] I previewed my documentation on a local server using `mkdocs serve`.
- [x] Running `mkdocs serve` didn't generate any failures.
- [x] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/experiment/sdks/android-sdk.md [hunk 1]
```diff
@@ -123,42 +146,69 @@ fun initialize(application: Application, apiKey: String, config: ExperimentConfi
...
+    | `pollOnStart` | Poll for local evaluation flag configuration updates once per minute on [start](#start). | `true` |
+    | `automaticFetchOnAmplitudeIdentityChange` | Only matters if you use the `initializeWithAmplitudeAnalytics` initialization function to seamlessly integrate with the Amplitude Analytics SDK. If `true` any change to the user ID, device ID or user properties from analytics will trigger the experiment SDK to fetch variants and update it's cache. | `false` |
+    | `userProvider` | An interface used to provide the user object to `fetch()` when called. See [Experiment User](https://developers.experiment.amplitude.com/docs/experiment-user#user-providers) for more information. | `null` |
+    | `exposureTrackingProvider` | Implement and configure this interface to track exposure events through the experiment SDK, either automatically or explicitly. | `null` |
+    | `instanceName` | Custom instance name for experiment SDK instance. **The value of this field is case-sensitive.** | `null` |
+
```
Comment (tyiuhc) [inline docs/experiment/sdks/android-sdk.md:185]: Missing `source` config

File: docs/experiment/sdks/android-sdk.md [hunk 2]
```diff
@@ -123,42 +146,69 @@ fun initialize(application: Application, apiKey: String, config: ExperimentConfi
...
+=== "Third Party Analytics"
 
     ```kotlin
-    val experiment = Experiment.initialize(context, "<DEPLOYMENT_KEY>", ExperimentConfig())
+    val experiment = Experiment.initialize(context, "DEPLOYMENT_KEY", ExperimentConfig())
     ```
 
+#### Configuration
+
+The SDK client can be configured once on initialization.
```
Comment (markzegarelli) [inline docs/experiment/sdks/android-sdk.md:165]:
```suggestion
SDK client configuration occurs during initialization.
```

File: docs/experiment/sdks/android-sdk.md [hunk 3]
```diff
@@ -123,42 +146,69 @@ fun initialize(application: Application, apiKey: String, config: ExperimentConfi
...
+    | `initialVariants` | An initial set of variants to access. This field is valuable for bootstrapping the client SDK with values rendered by the server using server-side rendering (SSR). | `{}` |
+    | `serverZone` | Select the Amplitude data center to get flags and variants from | `ServerZone.US` |
+    | `serverUrl` | The host to fetch remote evaluation variants from. For hitting the EU data center, use `serverZone`. | `https://api.lab.amplitude.com` |
+    | `flagsServerUrl` | The host to fetch local evaluation flags from. For hitting the EU data center, use `serverZone`. | `https://flag.lab.amplitude.com` |
+    | `fetchTimeoutMillis` | The timeout for fetching variants in milliseconds. | `10000` |
+    | `retryFetchOnFailure` | Whether to retry variant fetches in the background if the request doesn't succeed. | `true` |
+    | `automaticExposureTracking` | If true, calling [`variant()`](#variant) will track an exposure event through the configured `exposureTrackingProvider`. If no exposure tracking provider is set, this configuration option does nothing.  | `true` |
```
Comment (markzegarelli) [inline docs/experiment/sdks/android-sdk.md:178]:
```suggestion
    | `automaticExposureTracking` | If true, calling [`variant()`](#variant) tracks an exposure event through the configured `exposureTrackingProvider`. If no exposure tracking provider is set, this configuration option does nothing.  | `true` |
```

File: docs/experiment/sdks/android-sdk.md [hunk 4]
```diff
@@ -123,42 +146,69 @@ fun initialize(application: Application, apiKey: String, config: ExperimentConfi
...
+    | `fetchOnStart` | If true, always [fetch](#fetch) remote evaluation variants on [start](#start). If false, never fetch on start. If null, dynamically determine whether to fetch on start. | `null` |
+    | `pollOnStart` | Poll for local evaluation flag configuration updates once per minute on [start](#start). | `true` |
+    | `automaticFetchOnAmplitudeIdentityChange` | Only matters if you use the `initializeWithAmplitudeAnalytics` initialization function to seamlessly integrate with the Amplitude Analytics SDK. If `true` any change to the user ID, device ID or user properties from analytics will trigger the experiment SDK to fetch variants and update it's cache. | `false` |
```
Comment (markzegarelli) [inline docs/experiment/sdks/android-sdk.md:181]:
```suggestion
    | `automaticFetchOnAmplitudeIdentityChange` | Only matters if you use the `initializeWithAmplitudeAnalytics` initialization function to seamlessly integrate with the Amplitude Analytics SDK. If `true` any change to the user ID, device ID or user properties from analytics triggers the experiment SDK to fetch variants and update it's cache. | `false` |
```

File: docs/experiment/sdks/android-sdk.md [hunk 5]
```diff
@@ -123,42 +146,69 @@ fun initialize(application: Application, apiKey: String, config: ExperimentConfi
...
+    | `userProvider` | An interface used to provide the user object to `fetch()` when called. See [Experiment User](https://developers.experiment.amplitude.com/docs/experiment-user#user-providers) for more information. | `null` |
+    | `exposureTrackingProvider` | Implement and configure this interface to track exposure events through the experiment SDK, either automatically or explicitly. | `null` |
+    | `instanceName` | Custom instance name for experiment SDK instance. **The value of this field is case-sensitive.** | `null` |
+
+!!!info "EU Data Center"
+    If you're using Amplitude's EU data center, configure the `serverZone` option on initialization to `ServerZone.EU`.
```
Comment (markzegarelli) [inline docs/experiment/sdks/android-sdk.md:187]:
```suggestion
    If you use Amplitude's EU data center, configure the `serverZone` option on initialization to `ServerZone.EU`.
```

File: docs/experiment/sdks/android-sdk.md [hunk 6]
```diff
@@ -264,35 +316,59 @@ If you use either Amplitude or Segment Analytics SDKs to track events into Ampli
...
-    | `automaticFetchOnAmplitudeIdentityChange` | Only matters if you use the `initializeWithAmplitudeAnalytics` initialization function to seamlessly integrate with the Amplitude Analytics SDK. If `true` any change to the user ID, device ID or user properties from analytics will trigger the experiment SDK to fetch variants and update it's cache. | `false` |
-    | `userProvider` | An interface used to provide the user object to `fetch()` when called. See [Experiment User](https://developers.experiment.amplitude.com/docs/experiment-user#user-providers) for more information. | `null` |
-    | `exposureTrackingProvider` | Implement and configure this interface to track exposure events through the experiment SDK, either automatically or explicitly. | `null` |
-    | `instanceName` | Custom instance name for experiment SDK instance. **The value of this field is case-sensitive.** | `null` |
+Start the SDK by getting flag configurations from the server and fetching remote evaluation variants for the user. The SDK is ready once the returned future resolves.
```
Comment (markzegarelli) [inline docs/experiment/sdks/android-sdk.md:329]:
```suggestion
The Experiment SDK starts when you get flag configurations from the server and fetch remote evaluation variants for the user. The SDK is ready after the returned future resolves.
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

File: docs/experiment/sdks/android-sdk.md [hunk 7]
```diff
@@ -64,7 +61,11 @@ dependencies {
         }
         ```
 
-    === "Kotlin"
+    === "Third Party Analytics"
+
+        4. [Initialize the experiment client](#initialize)
+        5. [Start the SDK with the user](#fetch)
```
Comment (tyiuhc) [inline docs/experiment/sdks/android-sdk.md:67]: should map to `#start`?

Comment (bgiori) [reply docs/experiment/sdks/android-sdk.md:67]:
```suggestion
        5. [Start the SDK with the user](#start)
```

File: docs/experiment/sdks/android-sdk.md (hunk 6 above)
Comment (bgiori) [reply docs/experiment/sdks/android-sdk.md:329]:
```suggestion
Start the Experiment SDK to get flag configurations from the server and fetch remote evaluation variants for the user. The SDK is ready once the returned future resolves.
```

File: docs/experiment/sdks/android-sdk.md [hunk 8]
```diff
@@ -123,42 +146,70 @@ fun initialize(application: Application, apiKey: String, config: ExperimentConfi
...
+
+SDK client configuration occurs during initialization.
+
+???config "Configuration Options"
+    | <div class="big-column">Name</div> | Description | Default Value |
+    | --- | --- | --- |
+    | `debug` | Enable additional debug logging within the SDK. Should be set to false in production builds. | `false` |
+    | `fallbackVariant` | The default variant to fall back if a variant for the provided key doesn't exist. | `{}` |
+    | `initialVariants` | An initial set of variants to access. This field is valuable for bootstrapping the client SDK with values rendered by the server using server-side rendering (SSR). | `{}` |
+    | `source` | The primary source of variants. Set the value to `Source.InitialVariants` and configure `initialVariants` to bootstrap the SDK for SSR or testing purposes. | `Source.LocalStorage` |
```
Comment (tyiuhc) [inline docs/experiment/sdks/android-sdk.md:173]: `Source.LOCAL_STORAGE` and `Source.INITIAL_VARIANTS`

File: docs/experiment/sdks/android-sdk.md [hunk 9]
```diff
@@ -223,7 +276,7 @@ If you use either Amplitude or Segment Analytics SDKs to track events into Ampli
         ExperimentConfig config = ExperimentConfig.builder()
             .exposureTrackingProvider(new SegmentExposureTrackingProvider(analytics))
             .build();
-        ExperimentClient experiment = Experiment.initialize(context, "<DEPLOYMENT_KEY>", config);
```
Comment (tyiuhc) [inline docs/experiment/sdks/android-sdk.md:226]: do we need to remove angle brackets for `<FLAG_KEY>` as well?

File: docs/experiment/sdks/android-sdk.md (hunk 8 above)
Comment (bgiori) [reply docs/experiment/sdks/android-sdk.md:173]: local storage is the default.

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**